modules/pensions/spec/lib/pensions/monitor_spec.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require 'logging/monitor'
```

Comment:
Do we need these new imports here? Looks like nothing else in the file changed

Reply:
Nope, good catch!